packages/manager/apps/pci-ai-notebooks/src/hooks/api/ai/notebook/useStartNotebook.hook.tsx

```diff
@@ -0,0 +1,37 @@
+import { useMutation, useQueryClient } from '@tanstack/react-query';
+import { useParams } from 'react-router-dom';
+import { AIError, NotebookData } from '@/data/api';
+import { startNotebook } from '@/data/api/ai/notebook/notebook.api';
+
+interface UseStartNotebook {
+  onError: (cause: AIError) => void;
+  onStartSuccess: () => void;
+}
+
+export function useStartNotebook({
+  onError,
+  onStartSuccess,
+}: UseStartNotebook) {
+  const queryClient = useQueryClient();
+  const { projectId } = useParams();
+  const mutation = useMutation({
+    mutationFn: (notebookInfo: NotebookData) => {
+      return startNotebook(notebookInfo);
+    },
+    onError,
+    onSuccess: () => {
+      onStartSuccess();
+      // Invalidate service list query to get the latest data
+      queryClient.invalidateQueries({
+        queryKey: [projectId, 'ai/notebook'],
+      });
```

Comment:
Should we also invalidate this query ?   const queryKey = [projectId, 'ai/notebook', notebookId];

Reply:
Seen together, invalidateQueries will invalidate all queries starting by ai/notebook